nox/sessions.py

```diff
@@ -177,6 +177,10 @@ def run(self, *args, env=None, **kwargs):
         if not args:
             raise ValueError("At least one argument required to run().")
 
+        if self._runner.global_config.install_only:
```

Comment:
since `install` just calls `run`, what keeps this from skipping install commands as well?